imports/plugins/included/product-variant/components/productGridItems.js

```diff
@@ -66,9 +66,9 @@ class ProductGridItems extends Component {
       if (this.props.isMediumWeight()) {
         return (
           <div className={`product-additional-images ${this.renderVisible()}`}>
-            {this.props.additionalMedia().map((media) => {
-              return <span key={media._id} className="product-image" style={{ backgroundImage: `url('${media.url({ store: "medium" })}')` }} />;
-            })}
+            {/* eslint-disable max-len */}
+            {this.props.additionalMedia().map((media) => <span key={media._id} className="product-image" style={{ backgroundImage: `url('${media.url({ store: "medium" })}')` }} />)}
```

Comment:
This seems like a step backwards in terms of readability imho.

Reply:
I agree. There are a lot of places where readability seems to go backwards with all these new linter rules.

What do we do here though? Leave it and pass the linter?